pkg/cmd/repo/create/create.go

```diff
@@ -625,14 +626,14 @@ func interactiveRepoCreate(isDescEmpty bool, isVisibilityPassed bool, repoName s
 
 func confirmSubmission(repoName string, repoOwner string, inLocalRepo bool) (bool, error) {
 	qs := []*survey.Question{}
-
 	promptString := ""
+	normalizedRepoName := normalizeRepoName(repoName)
 	if inLocalRepo {
-		promptString = `This will add an "origin" git remote to your local repository. Continue?`
+		promptString = fmt.Sprintf(`This will create "%s" on GitHub and add it as the "origin" git remote. Continue?`, normalizedRepoName)
```

Comment:
Let's use the `OWNER/REPO` format in this prompt rather than just `REPO` since the `else` branch in this code also uses the `OWNER/REPO` format for the repository in the prompt. Rule of thumb is: repositories should always be displayed in `OWNER/REPO` format to reduce ambiguity.